src/components/landing-page/landing-page.js

```diff
@@ -13,9 +13,9 @@ class LandingPage extends Component {
     return (
       <div>
         LOGO HERE
-        <form>
+        <form className="searchTerm-form">
           <input id="searchTerm" type="text" placeholder="What do you need help with?"/>
-          <button onClick={this.handleClick.bind(this)}>?</button>
+          <button className="searchTerm-button" onClick={this.handleClick.bind(this)}>?</button>
```

Comment:
you don't need to bind 'this' because you are already in a class component and this.handleClick will refer to the class's function handleClick.